### PR TITLE
Added default data for ortho

### PR DESCRIPTION
**Pull request by hkirat**
In reference to #1623

### CODE REVIEW

File: src/webgl/camera.js [hunk 1]
```diff
@@ -141,6 +141,12 @@ p5.prototype.perspective = function(fovy,aspect,near,far) {
  *
  */
 p5.prototype.ortho = function(left,right,bottom,top,near,far) {
+  left = arguments[0] = arguments[0] || (-this.width/2);
```
Comment (lmccart) [inline src/webgl/camera.js]: these could be written more simply as:
```left = left || (-this.width/2);```

Comment (hkirat) [inline src/webgl/camera.js]: Might create a problem later on if I do this
https://github.com/processing/p5.js/blob/master/src/webgl/camera.js#L146

**Comment by lmccart** [comment]
ah I see. I've just removed all the validateParameters code so I think we can go with the simpler `left = left || (-this.width/2);`

**Comment by hkirat** [comment]
There might be another problem now.
If I pass a string as a parameter, 
left = left || (-this.width/2);
will retain the string parameter.
This might create a problem later on.

**Comment by lmccart** [comment]
only numbers are intended to be passed into this function. we may attempt parameter type validation again at a later date but for now it was too sporadic, buggy, and slowing down the code so I had to remove it until someone has the bandwidth to give it a focused look. it is fine to proceed with this function with the assumption only numbers will be passed in.

**Comment by hkirat** [comment]
Great!
I'll modify the code and remove the merge conflicts.

**Comment by hkirat** [comment]
Should be good to merge @lmccart

**Comment by lmccart** [comment]
wonderful, thank you

**Comment by jpsxlr8** [comment]
.

**Pull request merged**